library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+from sys import api_version
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka
+
+short_description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Red Hat OpenShift Streams for Apache Kafka Instance
```

Comment:
I would put link to our docs initially. Without reference those could be hard to decipher
https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview